java/ql/src/Security/CWE/CWE-489/DebuggableAttributeEnabled.qhelp

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When a debugger is enabled it could allow for entry points in the application or reveal sensitive information.</p>
+
+</overview>
+<recommendation>
+
+<p>In Android applications either set the <code>android:debuggable</code> attribute to <code>false</code>
+or do not include it in the manifest. The default value when not included is <code>false</code>.</p>
+
+</recommendation>
+<example>
+
+<p>In the example below, the <code>android:debuggable</code> attribute is set to <code>true</code>.</p>
+
+<sample src="DebuggableTrue.xml" />
+
+<p>The corrected version sets the <code>android:debuggable</code> attribute to <code>false</code>.</p>
+
+<sample src="DebuggableFalse.xml" />
+
+</example>
+<references>
+
+<li>
+  Android Developers:
+  <a href="https://developer.android.com/guide/topics/manifest/application-element#debug">The android:debuggable attribute</a>.
+</li>
```

Comment:
We could also add a reference about how enabling debug mode works in Android: https://developer.android.com/studio/debug#enable-debug

Reply:
Fixed in commit [49beb1e](https://github.com/github/codeql/pull/9939/commits/49beb1e716a29ff984ce943ac54555fa8be5f4a8).

Reply:
@atorralba I also added a reference the [App Manifest Overview](https://developer.android.com/guide/topics/manifest/manifest-intro) in commit [17d99ff](https://github.com/github/codeql/pull/9939/commits/17d99ff86c2d4d3f754e49dc9342e7775fc6cc0f). Let me know if you think this should be included or not.